Source/WebKit/Shared/RemoteLayerTree/RemoteLayerBackingStore.h

```diff
@@ -91,6 +93,10 @@ class RemoteLayerBackingStore : public CanMakeWeakPtr<RemoteLayerBackingStore> {
     virtual bool isRemoteLayerWithRemoteRenderingBackingStore() const { return false; }
     virtual bool isRemoteLayerWithInProcessRenderingBackingStore() const { return false; }
 
+    enum class ProcessModel : uint8_t { InProcess, Remote };
+    virtual ProcessModel processModel() const = 0;
+    static ProcessModel processModelForLayer(PlatformCALayerRemote*);
```

Comment:
Why is this function defined here. Can't be a non static method of `PlatformCALayerRemote`?

Reply:
Maybe? It's really more of an RLBS thing ("which kind of RLBS do you need to construct for this layer") IMO. It's factored out of the create function; I kind of like that they're all together :P